openupgrade_scripts/scripts/base/15.0.1.3/pre-migration.py

```diff
@@ -47,6 +47,28 @@
     ("l10n_ec.state_ec_24", "base.state_ec_24"),
 ]
 
+
+def fill_point_of_sale_refunded_orderline_id_column(cr):
+    # fill pos_order_line field
+    openupgrade.logged_query(
+        cr,
+        """
+        ALTER TABLE pos_order_line
+        ADD COLUMN IF NOT EXISTS refunded_orderline_id INTEGER
```

Comment:
I'd say it would be enough renaming the fields: `returned_line_id` and `refund_line_ids` to their new pairs in `point_of_sale`

Reply:
Understood